src/openforms/forms/admin/form.py

```diff
@@ -235,12 +236,14 @@ def get_queryset(self, request):
         )
 
     @admin.display(description=_("Actions"))
-    def get_object_actions(self, obj) -> str:
-        links = ((obj.get_absolute_url(), _("Show form")),)
+    def get_object_actions(self, obj: Form) -> str:
+        links: list[tuple[str, StrOrPromise]] = []
+        if obj.active:
+            links.append((obj.get_absolute_url(), _("Show form")))
```

Comment:
The link could also be made disabled: I made a draft implementation but this adds complexity to the code so let me know if it is necessary.

Reply:
this is fine!